Gruntfile.js

```diff
@@ -213,7 +213,11 @@ module.exports = function(grunt) {
     }))
 
     grunt.registerTask('build', 'Build project(s)css, templates, js', ['sass', 'browserify:ui', 'browserify:background', 'browserify:backgroundTest', 'execute:preProcessLists', 'safari'])
-    grunt.registerTask('dev', 'Build and watch files for development', ['build', 'watch'])
+
+    let devTasks = ['build']
```

Comment:
I keep seeing that we are using `let` almost exclusively (in this and all other PRs). Is there any reason for that? `const`s can save us from some nasty bugs and make it a bit easier (IMO) to read the code (by suggesting that given value will or will not be soon overwritten). Some more info: https://eslint.org/docs/rules/prefer-const

Reply:
This is a good question. I guess we don't have strict guidelines on `let` vs `const`.

Personally, I've tended to use `let` for objects/arrays to mean that they'll be mutated in the future, but thinking about it more, it's not ideal (and others [seem to agree](https://stackoverflow.com/questions/46553062/what-is-the-preferred-declaration-convention-for-objects-or-arrays-const-or-let)).

I'll switch to `const` here, please feel free to point out places in other PRs where I forget to do that!